pkg/features/README.md

```diff
@@ -0,0 +1,45 @@
+# Feature Flags
+
+This package provides a central point of truth for the codebase, indicating which features are available for execution.
+It determines feature availability based on the installed version of the linuxptp package.
+Integrating this package allows for conditional code execution depending on the enabled features.
+
+## Example
+
+```go
+import (
+    "github.com/k8snetworkplumbingwg/linuxptp-daemon/pkg/features"
+)
+
+func New(
+    ...
+) *Daemon {
+    ...
+    features.Flags.Init(getLinuxPTPPackageVersion())
+    features.Flags.Print()
+    ...
+}
+
+func (dn *Daemon) applyNodePtpProfile(runID int, nodeProfile *ptpv1.PtpProfile) error {
+    ...
+    if section.sectionName == "[global]" {
+        section.options["message_tag"] = messageTag
+        if socketPath != "" {
+            section.options["uds_address"] = socketPath
+        }
+
+        if feature.Flags.IsGMAvailable() { // Only runs if Grand master is enabled
```

Comment:
probably not a good example, configs parsing is done by looking what is given in ptpconfig
just have empty method
if feature.Flags.IsGMAvailable() { // Only runs if Grand master is enabled